texar/torch/modules/tokenizers/pretrained_bert_tokenizer_utils.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utils of pre-trained BERT tokenizer.
```

Comment:
Are components of this file written in reference to `pytorch-transformers`? We should probably give them credit here.

Reply:
Good point! I will add it accordingly in our implementations.